src/components/confirmOrderComponents/AreYouSure.js

```diff
@@ -73,31 +74,54 @@ function AreYouSure({showState,setShowState,portfolioId,stockId,buyOrSell,orderT
     }
 
     return(
-        <Modal centered show={showState} onHide={handleClose}>
-        <Modal.Header closeButton>
-            <Modal.Title>Please Review Your Trade</Modal.Title>
-        </Modal.Header>
+        <Modal centered show={showState} onHide={handleClose} backdrop="static">
+            <Row md={1} sm={1} xs={1} className="textCenter pt-1">
+                <Modal.Title className="bolded">{stockName}</Modal.Title>
+                <h5 className="newSpendingPowerSubtitle">New Spending Power {parseFloat(newPortfolioBalance).toLocaleString('en-US', {style: 'currency', currency: 'USD'})}</h5>
+            </Row>
         <Modal.Body>
             {error && <MessageAlert variant="danger">{error}</MessageAlert>}
-            {success && <MessageAlert variant="success">{success}</MessageAlert>}
+            {/* {success && <MessageAlert variant="success">{success}</MessageAlert>} */}
```

Comment:
Actually meant to say to take out this commented out code here.

Reply:
Fixed that there cheers!